packages/core/src/shared/remoteSession.ts

```diff
@@ -175,6 +181,61 @@ export async function handleMissingTool(tools: Err<MissingTool[]>) {
     )
 }
 
+function getFormattedSsmActions() {
+    const formattedActions = minimumSsmActions.map((action) => `"${action}",\n`).reduce((l, r) => l + r)
+
+    return formattedActions.slice(0, formattedActions.length - 2)
+}
+
+async function addInlinePolicyWithDelay(client: IamClient, roleArn: string) {
```

Comment:
a short docstring would be helpful, like:

/** Shows a message for X seconds then does Y if user cancels, else does Z. */